src/libraries/tests.proj

```diff
@@ -338,8 +338,6 @@
     <ProjectExclusions Include="$(MSBuildThisFileDirectory)Microsoft.Extensions.Logging.Abstractions\tests\Microsoft.Extensions.Logging.Generators.Tests\Microsoft.Extensions.Logging.Generators.Roslyn4.0.Tests.csproj" />
     <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Runtime.InteropServices\tests\LibraryImportGenerator.UnitTests\LibraryImportGenerator.Unit.Tests.csproj" />
     <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Runtime.InteropServices\tests\ComInterfaceGenerator.Unit.Tests\ComInterfaceGenerator.Unit.Tests.csproj" />
-    <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Text.Json\tests\System.Text.Json.SourceGeneration.Tests\System.Text.Json.SourceGeneration.Roslyn3.11.Tests.csproj" />
```

Comment:
These tests kept crashing for me in my RuntimeType PR with symptoms that suggested fatal OOMs. Example: https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/456077/logs/69

I have disabled them thinking that they use Roslyn, but it is not actually the case.

Reply:
They passed the outerloop test on the first commit of this PR. We get rather random OOMs in libs tests in general from time to time on Linux only. To fix it, we'd probably either need to drop parallelism or use the workstation GC, like we do for coreclr pri0 tests. I discussed it with Andy and the plan is to do something once it starts looking too severe.

Yep, these are the tests that the Json team fixed up for Native AOT in .NET 8 timeframe. The naming scheme always confuses me too (is this testing output of the source generator, or the source generator itself?).